### PR TITLE
refactor(common): post-review image directive cleanup

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
Minor cleanups based on the review comments that were
not addressed after the initial review.

Based on https://github.com/angular/angular/pull/47082#pullrequestreview-1074480563

### CODE REVIEW

File: packages/common/src/directives/ng_optimized_image/url.ts [hunk 1]
```diff
@@ -24,12 +24,6 @@ export function imgDirectiveDetails(rawSrc: string, includeRawSrc = true) {
   return `The NgOptimizedImage directive ${rawSrcInfo}has detected that`;
 }
```
Comment (gkalpak) [inline packages/common/src/directives/ng_optimized_image/url.ts:25]: The `imgDirectiveDetails()` function can also be removed, as it is also defined in [error_helper.ts][1].

[1]: https://github.com/angular/angular/blob/7516bebcd0ffeba45cc14c8e306b988bc71b62be/packages/common/src/directives/ng_optimized_image/error_helper.ts#L10-L14

**Review by gkalpak — COMMENTED** [review]
FYI, I have left a couple more minor comments in https://github.com/angular/angular/pull/47082#pullrequestreview-1075647565.

**Comment by pkozlowski-opensource** [comment]
Thnx for the additional review @gkalpak - I've converted this PR to a draft to capture your findings.

File: packages/common/src/directives/ng_optimized_image/url.ts (hunk 1 above)
Comment (pkozlowski-opensource) [reply packages/common/src/directives/ng_optimized_image/url.ts:25]: Good catch, thnx!

**Review by kara — APPROVED** [review]
LGTM

**Review by gkalpak — APPROVED** [review]

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 634e6c7f1ad9492f7ed6cec7aae2dfc132a704a1.

**Pull request closed without merging**